src/App.js

```diff
@@ -1,24 +1,46 @@
 import './App.scss';
+import 'toastr/build/toastr.min.css'
+import React, { useState } from 'react'
 import 'bootstrap/dist/css/bootstrap.min.css'
 import { HashRouter as Router, Switch, Route } from "react-router-dom";
-import {Entry, Dashboard, Account} from './pages'
-import {CreateRecipe, RecipeFeed, SingleRecipe} from './pages/RecipePages'
-import {CreateMealPlan, MealPlanFeed, SingleMealPlan} from './pages/MealPlanPages'
+import { Entry, Dashboard, Account } from './pages'
+import { CreateRecipe, RecipeFeed, SingleRecipe } from './pages/RecipePages'
+import { CreateMealPlan, MealPlanFeed, SingleMealPlan } from './pages/MealPlanPages'
+import ProtectedRoute from './components/ProtectedRoute'
+import ApiClient from './assets/apiClient'
+
 
 function App() {
+  // shows whether the user is or isn't logged in
+  const [isLoggedIn, isLoggedInHandler] = useState(false);
+  let apiClient = new ApiClient();
+
+
+  let logInFunc = (userToken) => {
+    isLoggedInHandler(true)
+    window.localStorage.setItem("UserToken", userToken);
+  }
+  
+  let logOutFunc = () => {
+    window.localStorage.removeItem("UserToken");
+    isLoggedInHandler(false)
+  }
+
+  // uncomment line 31 to logout current use
+  window.localStorage.removeItem("UserToken")
```

Comment:
This line is not commented out. Should it be?

Reply:
this is essentially the logout function without having a logout function - when it is uncommented you can log in and then uncomment to log out. i used it so i could test the login function